### ISSUE TITLE
Web API calls are not authenticated properly after entering Citra account.

### ISSUE THREAD

**Issue by chris062689** [citra-emu/citra#3339]
Steps to Reproduce:

Operating System: Linux
Bug Reported on Version: e05aa46 (This probably has been happening since initial release)

1. Clear out your citra-qt.ini file, open a fresh copy of Citra.
2. Fill in your Citra username / token. 
3. Verification will succeeded. 
4. Upon submitting telemetry or testcases, user will not be authenticated. 
5. Restart the Citra binary. 
6. Calls will now be authenticated properly. 

tl;dr Web API calls fail to authenticate when a user enters their username / token until they restart.

**Comment by FearlessTobi** [comment]
This issue is still present in the latest builds of Citra. What makes it even worse is, that when you try to submit testcases it shows you a "submission failed" popup without any proper reason.
